sklego/preprocessing.py

```diff
@@ -174,6 +175,10 @@ class ColumnSelector(BaseEstimator, TransformerMixin):
     """
 
     def __init__(self, columns: list):
+        # if the columns parameter is not a list, make it into a list
+        if not isinstance(columns, list):
```

Comment:
this is a detail. but wouldn't this be a usecase for the `as_list` function we have in `sklego.common`?

Reply:
This is my only issue now.

Reply:
@koaning I just readed `as_list` function and I agree with you. I did the changes you can now review them.